crates/store/re_chunk/src/lib.rs

```diff
@@ -37,11 +37,12 @@ pub use self::batcher::{
 // Re-exports
 
 #[doc(no_inline)]
-pub use arrow::array::Array as ArrowArray;
-#[doc(no_inline)]
-pub use re_log_types::{EntityPath, TimeInt, TimePoint, Timeline, TimelineName};
-#[doc(no_inline)]
-pub use re_types_core::{ArchetypeFieldName, ArchetypeName, ChunkId, ComponentName, RowId};
+pub use {
+    arrow::array::Array as ArrowArray,
+    re_log_types::{EntityPath, TimeInt, TimePoint, Timeline, TimelineName},
+    re_span::Span,
```

Comment:
I'm not fan of this re-export. It has the potential to confuse the LSP (and the dev) with a toss of coins whether `re_span::Span` or `re_chunk::Span` gets included. I'd much rather have `re_chunk` re-export `external::re_span` instead.